NEWS

```diff
@@ -1,22 +1,45 @@
-x/y/2022 ola-0.10.9
+x/y/2023 ola-0.10.9
  Features:
- *
+ * Python 3 compatibility across the board (including the RDM Responder Tests)!
+   #1506
+ * Support for the JMS USB2DMX PRO V2.1 device via the FTDI plugin #1728
 
  API:
  * Python: Add a fetch current DMX example.
 
  RDM Tests:
- * 
+ * Python 3 compatibility of the RDM Tests #1599
```

Comment:
Also not merged yet but same as above: will be merged before 0.10.9 is released

Reply:
Again.

Reply:
Again reverted in https://github.com/OpenLightingProject/ola/pull/1761/commits/c41589de0b67015f5d94eafb061eba511801ef69